lib/src/components/Drawer/AssetDrawer/index.tsx

```diff
@@ -15,12 +15,24 @@ export const AssetDrawerComponent = forwardRef<'div', AssetDrawerProps>(
   ({ children, maxWidth = 820, store, ...rest }, ref) => {
     const theme = useTheme()
 
+    const getPersistentElements = () =>
+      Array.from(document.querySelectorAll('[data-wui-persistent]'))
+
+    const hideOnInteractOutside = (event: Event) => {
+      const target = event.target as HTMLElement
+      const isTargetWithinPersistentElements = getPersistentElements().some(element =>
+        element.contains(target)
+      )
+      return !isTargetWithinPersistentElements
+    }
+
     return (
       <Drawer
         {...rest}
         autoFocusOnShow={false}
+        getPersistentElements={getPersistentElements}
```

Comment:
We pass a `getPersistentElements` function already from `wttj-front`. Maybe we should put the `{...rest}` at the end of the props to allow overrides ?

Reply:
I updated the code so the behavior is the same as prior to this PR when you pass a `getPersistentElements()` prop